ensembling/qra-ensemble-sum-of-states/update-sum-of-states-qra-ensemble.R

```diff
@@ -0,0 +1,59 @@
+
+library(magrittr); library(ggplot2)
```

Comment:
missing here,  dplyr, readr, tidyr, cowplot and data.table - though all functions from these packages are called using  package:fn so doesn't make a great deal of difference.